Menu/Menu.js

```diff
@@ -1,10 +1,22 @@
 
+
+
+
 const toggleMenu = () => {
+  menu.classList.toggle("menu--open");
   // Toggle the "menu--open" class on your menu refence. 
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector(".menu-button");
 // Using your menuButton reference, add a click handler that calls toggleMenu
+
+menuButton.addEventListener('click', e => {
```

Comment:
```suggestion
menuButton.addEventListener('click', toggleMenu)
```
If you're not declaring an event listener inside a class constructor you don't need to pass your function into an anonymous callback. You can just reference the function you wrote and when the click occurs it will fire the function invocation.